go.mod

```diff
@@ -1,15 +1,17 @@
 module github.com/confidential-containers/trustee-operator
 
-go 1.21
+go 1.22
```

Comment:
You'll need to use go 1.21 in your dev env to make this update. 
The 1.36.1 sdk is still on 1.21. Although not strictly needed, but still it's good to use the same golang version as used with the sdk.

Reply:
ok, will revert back to go 1.21

Reply:
Not sure I should revert to go 1.21 though. When running` go mod tidy` the golang version is updated automatically to go 1.22.
This is because of the following dep:

```
go mod tidy -go=1.21 -compat=1.21
go: github.com/google/pprof@v0.0.0-20240827171923-fa2c70bbbfe5 requires go@1.22, but 1.21 is requested

```

Reply:
Hmm and do you know what is using pprof ? You will need to downgrade pprof otherwise.

Reply:
pprof comes as an indirect dependency. The weird thing is that all the direct deps are aligned to https://github.com/operator-framework/operator-sdk/blob/v1.36.1/go.mod

Reply:
Ok, one trick I use is to explicitly download a specific version of the offending dep. For example using the commit 
before fa2c70bbbfe53334fe0fd4c80336583970707379 (https://github.com/google/pprof/commits/main/) should fix it imho.
But if you hit additional issues, then we'll have to stick to 1.22. Build and test should help uncover any potential issues.

Reply:
It seems the trick works ok, Thank you